### PR TITLE
added cloudwatch enable/disable actions

### PR DESCRIPTION
**Pull request by steffyP**
* added `enable-alarm-actions` and `disable-alarm-actions` 
   *  fix for https://github.com/localstack/localstack/issues/5562
* added snapshot test
* fixed and improved some return values in cloudwatch to match AWS responses

### CODE REVIEW

File: localstack/services/cloudwatch/alarm_scheduler.py [hunk 1]
```diff
@@ -28,8 +29,7 @@
 STATE_OK = "OK"
 STATE_INSUFFICIENT_DATA = "INSUFFICIENT_DATA"
```
Comment (silv-io) [inline localstack/services/cloudwatch/alarm_scheduler.py:30]: These values are also in the generated ASF API: `localstack.aws.api.cloudwatch.StateValue`

File: localstack/services/cloudwatch/alarm_scheduler.py [hunk 2]
```diff
@@ -276,37 +284,58 @@ def calculate_alarm_state(alarm_arn: str) -> None:
...
     client = get_cloudwatch_client_for_region_of_alarm(alarm_arn)
 
+    query_date = datetime.utcnow().strftime(format="%Y-%m-%dT%H:%M:%S+0000")
     metric_values = collect_metric_data(alarm_details, client)
 
+    state_reason_data = {
+        "version": "1.0",
+        "queryDate": query_date,
+        "period": alarm_details["Period"],
+        "recentDatapoints": [v for v in metric_values if v],
```
Comment (silv-io) [inline localstack/services/cloudwatch/alarm_scheduler.py:294]: This would remove values==0. Is this intended?

**Review by silv-io — APPROVED** [review]
LGTM! Just some really minor nitpicks 😄

File: localstack/services/cloudwatch/alarm_scheduler.py (hunk 2 above)
Comment (steffyP) [reply localstack/services/cloudwatch/alarm_scheduler.py:294]: no, not intended - thanks for pointing this out 🙏

**Pull request merged**